tests/integration/backward_compatible/proxy/map_test.py

```diff
@@ -796,6 +796,14 @@ def setUp(self):
     def tearDown(self):
         self.map.destroy()
 
+    def test_aggregate_with_none_aggregator(self):
+        with self.assertRaises(AssertionError):
+            self.map.aggregate(None)
```

Comment:
Can be simplified as: `self.assertRaises(AssertionError, lambda: self.map.aggregate(None))` I think most of other assertRaises uses can also be put into a single line.

Reply:
I personally prefer this style, and try to use it like this across the codebase. Advantages:

- You can easily put breakpoints
- You can put multiple lines without defining a closure

And IMHO, this one is more readable